Automattic/wp-calypso
# Jetpack Cloud: Get Scan status from API

#### Changes proposed in this Pull Request

- Create a Query Component to fetch the Scan status from the API and load the data into the store.

#### Testing instructions

- Apply the "Add dummy scan api endpoint" patch to your sandbox (D41270-code)
- Navigate to http://calypso.localhost:3000
- http://jetpack.cloud.localhost:3000/scan/<siteSlug>?scan-state=threats
- See in the network tab the request to `/site/<siteId>/scan`
- Check the JETPACK_SCAN_REQUEST action has been dispatched
- Check in Redux that the data is stored in `jetpackScan.scan` and the request status in `jetpackScan.requestStatus`

See 1151678672052943-as-1169562672652097.
